nevergrad/benchmark/experiments.py

```diff
@@ -1197,11 +1197,140 @@ def images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Optimizing images: artificial criterion for now."""
+    seedg = create_seed_generator(seed)
+    optims = ["CMA", "NGOpt8", "DE", "PSO", "RecES", "RecMixES", "RecMutDE", "ParametrizationDE"]
+    if default_optims is not None:
+        optims = default_optims
+    funcs:tp.List[ExperimentFunction] = [imagesxp.Image(loss=loss) for loss in [
+                imagesxp.imagelosses.SumAbsoluteDifferences,
+                imagesxp.imagelosses.LpipsAlex,
+                imagesxp.imagelosses.LpipsVgg,
+                imagesxp.imagelosses.SumSquareDifferences,
+                imagesxp.imagelosses.HistogramDifference]]
+    base_values: tp.List[tp.Any] = [func(func.parametrization.sample().value) for func in funcs]
+    mofuncs: tp.List[tp.Any] = fbase.multi_experiments(funcs, upper_bounds=base_values, pareto_size=25) if cross_valid else [
+            fbase.MultiExperiment(funcs, upper_bounds=base_values)]
+    for budget in [100 * 5 ** k for k in range(3)]:
+        for num_workers in [1]:
+            for algo in optims:
+                for mofunc in mofuncs:
+                    xp = Experiment(mofunc, algo, budget, num_workers=num_workers, seed=next(seedg))
+                    yield xp
+
+
+@registry.register
+def image_multi_similarity_cv(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of image_multi_similarity with cross-validation."""
+    return image_multi_similarity(seed, cross_valid=True)
+
+
+
+@registry.register
+def image_quality_proxy(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Optimizing images: artificial criterion for now."""
+    seedg = create_seed_generator(seed)
+    optims: tp.List[tp.Any] = get_optimizers("structured_moo")
+    if default_optims is not None:
+        optims = default_optims
+
+    
+    # We optimize func_blur or func_brisque and check performance on func_iqa.
+    func_iqa = imagesxp.Image(loss=imagesxp.imagelosses.Koncept512)
+    func_blur = imagesxp.Image(loss=imagesxp.imagelosses.Blur)
+    func_brisque = imagesxp.Image(loss=imagesxp.imagelosses.Brisque)
+    
+    # TODO: add the proxy info in the parametrization.
+    for budget in [100 * 5 ** k for k in range(3)]:
+        for num_workers in [1]:
+            for algo in optims:
+                for func in [func_blur.copy(), func_brisque.copy()]:
+                    func.special_evaluation_function = func_iqa.evaluation_function
+                    xp = Experiment(func, algo, budget, num_workers=num_workers, seed=next(seedg))
+                    yield xp
+
+
+@registry.register
+def image_quality(seed: tp.Optional[int] = None, cross_val: bool=False) -> tp.Iterator[Experiment]:
+    """Optimizing images for quality:
+    TODO
+    """
+    seedg = create_seed_generator(seed)
+    optims: tp.List[tp.Any] = get_optimizers("structured_moo")
+    if default_optims is not None:
+        optims = default_optims
```

Comment:
I still do not like this, we can have a switch that overrides get_optimizers instead (for later)

Reply:
Right, this looks like a good idea shortening everything. Can do after the present PR.